firmware/hw_layer/sensors/CJ125.cpp

```diff
@@ -290,8 +290,19 @@ static void cjStart(void) {
 		return;
 	}
 	
+	// Software reset cj125
+	for (int i = 0; i < 2; i++)
+	{
+		cjWriteRegister(INIT_REG2_WR, CJ125_INIT2_RESET)
+		chThdSleepMilliseconds(100);
+	}
+
 	cjIdentify();
 
+	// default values
```

Comment:
please add code comment clarifying when would these values be used? looks like mandatory calibration follows this code?